README.md

```diff
@@ -2,7 +2,7 @@
 
 This application is a simple PHP application using the package `cron/cron` installed from Composer
 
-Application → https://sample-php-cron.scalingo.io/
+Application → https://php-cron.is-easy-on-scalingo.com
 
```

Comment:
question: I checked https://sample-php-cron.scalingo.io/ redirects to https://php-cron.is-easy-on-scalingo.com which is fine ^^; how to know about which such URLs are the old ones and the new ones to use ?

Reply:
The new one is the one defined as he canonical domain on the Scalingo applcation